lib/ardb/has_slug.rb

```diff
@@ -53,11 +53,12 @@ def reset_slug
 
       def ardb_has_slug_generate_slug
         slug_attr_value = self.send(self.class.ardb_has_slug_config[:attribute])
-        if !slug_attr_value || slug_attr_value.to_s.empty?
-          slug_attr_value = self.instance_eval(&self.class.ardb_has_slug_config[:source_proc])
+        slug_source     = slug_attr_value
```

Comment:
@jcredding I don't understand why you are calling this var the slug source.  In the prev implementation it was called `slug_string`.  I think this is a better name as it may or may not be the source.  In some cases it is the current attr value and in others it is the result of the source.